### DISCUSSION TITLE
How to resize and align my images?

### DISCUSSION THREAD

**Discussion by carlos-truong** [ObsidianPublisher/obsidian-github-publisher#316] [Q&A — Plugin]
Hi,

I'm trying to resize my images like this:
- ![[Attachments/discord.png|100]]

I've created a site to test it:
https://vu-truong.github.io/

But it doesn't work and it seems that the size becomes the title of img tag:
- `<img alt="100" src="Attachments/discord.png" title="100">`

So, is there any way to resize the images?
Besides that, how to align these images to the right, center, left?

**Comment by Mara-Li** [comment]
Did you use the old template? Because everything should works with it.

As I stated in the last announcement, I will drop support for mkdocs. It takes too much of my time, and is a pain in the ass to works with.

**Reply by carlos-truong** [reply]
Hi Lisandra,
Thank you for your support. It helps me a lot!

I just created a repo to test it and I've run the "get latest release from template". Here is my repo:
https://github.com/vu-truong/vu-truong.github.io

I also don't understand why it does not work. Is it because of the template?

**Reply by Mara-Li** [reply]
It should works with the last version of the template and the Javascript used. 

Sorry but I didn't sleep this night, I can't help more today.

**Reply by Mara-Li** [reply]
Should be fixed in some hour, times to jsdelivr to update the cache.

**Reply by carlos-truong** [reply]
Thank you for your support :D
Besides resizing, could I align my images to the right, left, center? I couldn't find any docs about it.

**Reply by Mara-Li** [reply]
https://obsidian-publisher.netlify.app/advanced/snippets%20and%20tools/#css-snippets

**Reply by Mara-Li** [reply]
You need to do it yourself using CSS and ALT attribute.

**Reply by carlos-truong** [reply]
Hi Lisandra,
Where should I put my snippets? 

And is it like Obsidian snippets?
I put this snippet to `docs/assets/css` but it does not work (it works in Obsidian)
```
img[alt*="center"] {
    display: block;
    margin-left: auto;
    margin-right: auto;
}
```

**Reply by Mara-Li** [reply]
I'm sorry you need to check it yourself.

**Reply by carlos-truong** [reply]
Okay! Thank you for your support :D
